projects/distributed-tracing/src/shared/dashboard/widgets/table/table-widget-renderer.component.ts

```diff
@@ -59,4 +60,8 @@ export class TableWidgetRendererComponent
   public get syncWithUrl(): boolean {
     return this.model.style === TableStyle.FullPage;
   }
+
+  public onRowSelection(selections: TableRow[]): void {
+    this.api.model.selectionHandler?.execute(selections[0]);
```

Comment:
This is weird. Since the output is same for both single and multi selection, at this point, how should we handle the action? Above change would work for single selection, but if someone enables multi selection on the table and there are multiple row selections, should we just ignore such a case? Also, selections[0] will not always be the latest selected row.

Reply:
So considering that when we click on a selected row it gets unselected, what should be the behavior of the sheet overlay? Should it also gets closed?  Should it stay open while the row selection style is removed?

Reply:
I'd think we'd just give the whole array to the selection handler - it's on the handler to deal with it.
As to close behavior, I'd say it should close either on any external click, or none - wouldn't special case the deselect case.

Reply:
I have decided to hide it as the row is un selected. Lemme know if you disagree

Reply:
@aaron-steinfeld that makes sense. Since I am using detail sheet handler, I can't put any selection specific code in the handler. 
However, I can put a restriction that if it gets empty or undefined data, then it would clear out its state/dispose any popover etc. I can see it work in that case. Do you think its a good assumption?

Reply:
Yes more or less - I'd have it take an array of values, and then the specific handler can choose how to react to that. This handler can close any popups if receiving an empty array, open/replace if receiving a single, and error if receiving multiple.

Reply:
nice. This works.